testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestAllDatatypesFromHiveConnector.java

```diff
@@ -24,6 +24,7 @@
 import io.trino.tempto.fulfillment.table.TableHandle;
```

Comment:
Fix cmt msg to: `Mark testSelectAllDatatypesParquetFile as @Flaky`

Reply:
it's "fixes" because it covers HS2 problem and we do not plan to do anything more with regards to this test.